src/blob_detection.cpp

```diff
@@ -304,7 +304,7 @@ namespace Blob_Detection
     const std::vector < BlobInfo > & BlobDetection::find( const PenguinV_Image::Image & image, uint32_t x, uint32_t y, uint32_t width,
                                                           uint32_t height, BlobParameters parameter, uint8_t threshold )
     {
-        Image_Function::ParameterValidation( image, x, y, width, height );
+        Image_Function::ParameterValidation( width, height, image, x, y );
```

Comment:
Why did we change input variables for this function? Now we have to refactor whole library for this

Reply:
Yep, I know, but the only way to make images and their dimensions variadic, was to put them at the end of the function..